test/ts-scenario/steps/lib/listeners.ts

```diff
@@ -17,13 +21,15 @@ import os = require('os');
 const stateStore: StateStore = StateStore.getInstance();
 const CHECKPOINT_FILE_KEY = 'checkpointFile';
 
-export async function createContractListener(gatewayName: string, channelName: string, ccName: string, eventName: string, listenerName: string, listenerOptions: ListenerOptions): Promise<void> {
+export async function createContractListener(gatewayName: string, channelName: string, ccName: string,
+	eventName: string, listenerName: string, listenerOptions: ListenerOptions): Promise<void> {
 	const gateways: Map<string, any> = stateStore.get(Constants.GATEWAYS);
 	const gateway: Gateway  = gateways.get(gatewayName).gateway;
 	const contract: Contract = await GatewayHelper.retrieveContractFromGateway(gateway, channelName, ccName);
 
 	const payloads: ContractEvent[] = [];
 
+	// eslint-disable-next-line @typescript-eslint/require-await
```

Comment:
Remove `async` and return `Promise.resolve()` to avoid lint error rather than disabling rule